### PR TITLE
Validate registry.suse.com containers as well

### PR DESCRIPTION
**Pull request by dirkmueller**
The signing key has changed and publishing was half broken. so lets test for it so that we find it immediately

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -326,6 +330,28 @@ jobs:
...
+          +hIxtmPOhqDKHH0As113sjTqqo20Ik233x9FFeTFD8Or7ahpqjiv5YCufk9AoQbC
+          xMIjrK9RkQYgW4RycgvXGASobwN8EE+OsMcyMUER/pdCtQhTQCc1jYLt85VhfEkC
+          4k9szMB8eZrdV9re/Ku6vnCeXRR5yn2NWKO88U4HfxEpJv5s2uFJi37+x/v9w7Uh
+          +864W/9NexXg/JFNsvh0Kmxsbi3ZegaouLyrMCHwSA3ByBZ2yCf2VuFPyUCNEZOH
+          Owi0oc9TgY1yopjsTneyGaMCAwEAAQ==
+          -----END PUBLIC KEY-----
+          EOF
+
+          cat /etc/containers/policy.json | jq .
+
```
Comment (dcermak) [inline .github/workflows/ci.yaml:354]: Debugging leftover?
```suggestion
```

**Comment by dcermak** [comment]
Looks like SP3 & SP4 have not yet been re-signed with the old key. If they won't be resigned, then we'll have to resort to a different key for them.

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (dirkmueller) [reply .github/workflows/ci.yaml:354]: yeah, this is still a draft..

**Comment by dirkmueller** [comment]
> Looks like SP3 & SP4 have not yet been re-signed with the old key. If they won't be resigned, then we'll have to resort to a different key for them.

the plan is to fix the signing. That's why I added tests. the tests can be merged when they're green to avoid regressing on this in the future.

**Comment by dirkmueller** [comment]
remaining failures are unrelated to this change

**Pull request merged**